manoseimas/mps_v2/models.py

```diff
@@ -74,16 +75,16 @@ def __unicode__(self):
 
 class Group(CrawledItem):
     GROUP_TYPES = (
-        ('group', ('Group')),
-        ('committee', ('Committee')),
-        ('commission', ('Commission')),
-        ('fraction', ('Fraction')),
-        ('parliament', ('Parliament')),
+        ('group', _('Group')),
+        ('committee', _('Committee')),
+        ('commission', _('Commission')),
+        ('fraction', _('Fraction')),
+        ('parliament', _('Parliament')),
```

Comment:
Apparently it was using `ugettext`, fixed in e593f753e60042fa49a1c7499407f282e35e5fe0.

Groups extracted into constants in 8d45dc5aceceecf72dd908004b53105714db9442.